.travis.yml

```diff
@@ -8,23 +8,25 @@ addons:
     packages:
     - g++-6
     - cmake
-    - python-dev
-    - python-numpy-dev
+    - python3-dev
+    - python3-numpy-dev
     - swig
     - libeigen3-dev
     - liblapack-dev
-    - python-matplotlib
+    - python3-matplotlib
 
 matrix:
   include:
     # In source build
     - script:
-      - cmake -DCMAKE_CXX_COMPILER=g++-6 .
+      - alias python=python3
+      - cmake -DCMAKE_CXX_COMPILER=g++-6 -DPYTHON_EXECUTABLE=`which python3` .
       - make
     # Out of source build
     - script:
+      - alias python=python3
       - mkdir build
       - cd build
-      - cmake -DCMAKE_CXX_COMPILER=g++-6 ../
+      - cmake -DCMAKE_CXX_COMPILER=g++-6 -DPYTHON_EXECUTABLE=`which python3` ../
```

Comment:
Instead of replacing Python 2 with Python 3, I would prefer to keep support for Python 2, at least for now. Hence, it would be good to make an additional Python 3 test here, instead of changing the old one.